module/proto/gravity/v1/attestation.proto

```diff
@@ -26,26 +26,30 @@ enum ClaimType {
   CLAIM_TYPE_VALSET_UPDATED      = 5;
 }
 
```

Comment:
I took the liberty of beefing up the comments in these proto files so that we can more closely approach a standard format